gem/oq-platform
# Custom geonode and geoserver ports.

This pull-request allows to install more than one platform on the same machine (one platform at time for each user).
To test it use as bootstrap command:


You can use:
  ```fab bootstrap:db_name=<your-dbname>,db_user=<your-dbuser>,geonode_port=<gn_port>,geoserver_port=<gs_port>```
  ```fab start:geonode_port=<port_gn>,geoserver_port=<port_gs>```
  ```fab stop```
  ```fab clean:db_name=<your-dbname>,db_user=<your-dbuser>```

or set ```GEM_DB_NAME```, ```GEM_DB_USER```, ```GEM_GEONODE_PORT``` and ```GEM_GEOSERVER_PORT``` environment  variables to inherit these parameters without explicitly pass them as fab arguments.

```stop``` or ```clean``` should not interfer with other platform installations.
